src/jobs/notify.yml

```diff
@@ -20,21 +20,21 @@ parameters:
   circle_token:
     description: >
       CircleCI token for fetching flaky tests via API.
+      If not specified, CIRCLE_TOKEN environment variable will be used.
       Please refer to this document how to generate it.
-      https://circleci.com/docs/managing-api-tokens
-    default: CIRCLE_TOKEN
-    type: env_var_name
+      https://circleci.com/docs/managing-api-tokens (requires a v2 token)
+    default: $CIRCLE_TOKEN
+    type: string
   channel:
     description: >
       Select which channel in which to post to.
       Channel name or ID will work.
       You may include a comma separated list of channels if you wish to post to multiple channels at once.
       Set the "SLACK_DEFAULT_CHANNEL" environment variable for the default channel.
-    default: SLACK_DEFAULT_CHANNEL
-    type: env_var_name
+    default: $SLACK_DEFAULT_CHANNEL
+    type: string
 
 steps:
-  - checkout
```

Comment:
Using this orb's job, I wouldn't expect it to be checking out my repo. Not sure if this was here for a specific or different use case?